.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea
```

Comment:
Why should IntelliJ project files be ignored?

Reply:
Yes but this is a VSC plugin so why should IntelliJ platform files be ignored for the whole plugins and why not e.g. Eclipse or NetBeans files?